docs/root/intro/arch_overview/data_sharing_between_filters.rst

```diff
@@ -0,0 +1,93 @@
+.. _arch_overview_data_sharing_between_filters:
+
+Envoy provides the following mechanisms for the transfer of configuration,
+metadata and per-request/connection state to, from and between filters, as
+well as to other core subsystems  (e.g., access logging).
+
+
+Static State
+============
+
+Static state is any immutable state specified at configuration load time
+(e.g., through XDS). There are three categories of static state:
+
+Metadata
+--------
+
+Several parts of Envoy configuration (e.g. listeners, routes, clusters )
+contain a metadata field (map<string, Protobuf::Struct>) where arbitrary
+key-value pairs can be encoded.  The typical pattern is to use the filter
+names in reverse DNS format as the key and encode filter specific
+configuration metadata in the value. This metadata is immutable and shared
+across all requests/connections. Such config metadata is usually provided
+during bootstrap time or as part of XDS. For example, weighted clusters in
```

Comment:
Please s/XDS/xDS/ everywhere to be consistent with the use of this TLS across Envoy. Thanks.